packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -31,8 +25,19 @@ import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
 import LeafIcon from '@patternfly/react-icons/dist/esm/icons/leaf-icon';
 import FolderIcon from '@patternfly/react-icons/dist/esm/icons/folder-icon';
 import FolderOpenIcon from '@patternfly/react-icons/dist/esm/icons/folder-open-icon';
+import SortAmountDownIcon from '@patternfly/react-icons/dist/esm/icons/sort-amount-down-icon';
 
 import { Checkbox, ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import {
+Toolbar,
+ToolbarItem,
+ToolbarContent,
+OptionsMenu,
+OptionsMenuItemGroup,
+OptionsMenuItem,
+OptionsMenuSeparator,
+OptionsMenuToggle
+} from '@patternfly/react-core';
```

Comment:
I don't think you need any of these imports.

Reply:
I thought I needed them because the table was a separate package, so any imports from core had to be specified at the header?